phpunit.xml.dist

```diff
@@ -9,19 +9,9 @@
          stopOnFailure="false"
          bootstrap="vendor/autoload.php"
     >
-    <php>
-        <ini name="date.timezone" value="UTC" />
-    </php>
-
     <testsuites>
         <testsuite name="XdebugHandler Test Suite">
             <directory>tests</directory>
         </testsuite>
     </testsuites>
-
-    <filter>
-        <whitelist>
-            <directory>src</directory>
```

Comment:
The whitelist config should not be removed, otherwise generating code coverage becomes impossible.

Reply:
Yeah, it's a bit brutal but the `<filter>` element has been replaced by `<coverage>` in PHPUnit 9.

You can still generate code coverage though with the appropriate command line option (`--whitelist <dir>` or `--coverage-filter <dir>`).

Reply:
Well, PHPUnit 9 still supports the old format